packages/session-replay-browser/src/config/local-config.ts

```diff
@@ -38,6 +44,7 @@ export class SessionReplayLocalConfig extends Config implements ISessionReplayLo
     this.serverZone = options.serverZone || DEFAULT_SERVER_ZONE;
     this.configEndpointUrl = options.configEndpointUrl;
     this.shouldInlineStylesheet = options.shouldInlineStylesheet;
+    this.version = options.version;
```

Comment:
Curious - what is session replay version? 

cc: @Mercy811 in case we need to do something similar with remote config for analytics too.

Reply:
It looks like it's type (enum) + version (string) https://github.com/amplitude/Amplitude-TypeScript/blob/e33f45a4f496cb0c01bec3d91137fc62d969ff67/packages/session-replay-browser/src/config/types.ts#L70-L75

Reply:
This is for us to send the library type (standalone, plugin, wrapper) and the version of that library.